cpg_workflows/inputs.py

```diff
@@ -146,7 +146,7 @@ def _populate_analysis(cohort: Cohort) -> None:
                     analysis.output,
                 )
                 sequencing_group.gvcf = GvcfPath(path=analysis.output)
-            elif exists(sequencing_group.make_gvcf_path()):
+            elif exists(sequencing_group.make_gvcf_path().path):
```

Comment:
Is this (and the similar change on line 165) a bug fix?

These things are a GvcfPath and CramPath respectively, which (despite the names) are not is-a-kind-of-Path 😢  so getting at their `path` member (which actually is-a-Path-or-str) is warranted.

Adding `.path` is probably clearer, though deep in the cloudpathlib guts of `exists()` there is an explicit `str(…)` which may well have made this work. Hence question: did you observe this code not firing — thus buggy behaviour — or is this just a clarification?

Reply:
<img width="863" alt="Screenshot 2024-04-03 at 1 40 39 PM" src="https://github.com/populationgenomics/production-pipelines/assets/5886855/62b20d60-df2b-49d9-9984-16a594d4b2a4">

This popped up in my IDE as a warning - the argument being provided is an invalid type, so I'm guessing either this somehow works on the string/.path representation, or we never exercise this code at runtime

Reply:
Interesting… So the remaining question is why mypy didn't complain about this at the time… 🤔